Athena query fails when fetchType = STORE

### Describe the issue

This sample flow, based on [the Athena unit test flow](src/test/java/io/kestra/plugin/aws/athena/QueryTest.java), fails with an exception when `fetchType: STORE`. Internet searching suggests this can be fixed by [appropriately configuring the Jackson ObjectMapper](https://www.baeldung.com/jackson-jsonmappingexception#1-globally-auto-detect-fields-with-any-visibility).

```
id: athena_serialization_fail
namespace: company.team

tasks:
  - id: athena_query
    type: io.kestra.plugin.aws.athena.Query
    region: eu-west-3
    database: units
    outputLocation: s3://kestra-unit-test
    fetchType: STORE
    query: |
      select * from types
```

The exception is:
```
TRACE 2025-11-07T12:01:58.932637Z reactor.core.Exceptions$ReactiveException: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class software.amazon.awssdk.services.athena.model.Row and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)
	at reactor.core.Exceptions.propagate(Exceptions.java:410)
	at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:102)
	at reactor.core.publisher.Mono.block(Mono.java:1779)
	at io.kestra.plugin.aws.athena.Query.store(Query.java:313)
	at io.kestra.plugin.aws.athena.Query.run(Query.java:249)
	at io.kestra.plugin.aws.athena.Query.run(Query.java:45)
	at io.kestra.worker.WorkerTaskCallable.doCall(WorkerTaskCallable.java:81)
	at io.kestra.worker.AbstractWorkerCallable.call(AbstractWorkerCallable.java:64)
	at io.kestra.worker.WorkerSecurityService.callInSecurityContext(WorkerSecurityService.java:10)
	at io.kestra.worker.DefaultWorker.lambda$callJob$19(DefaultWorker.java:972)
	at io.kestra.core.trace.NoopTracer.inCurrentContext(NoopTracer.java:15)
	at io.kestra.worker.DefaultWorker.callJob(DefaultWorker.java:968)
	at io.kestra.worker.DefaultWorker.runAttempt(DefaultWorker.java:926)
	at io.kestra.worker.DefaultWorker.run(DefaultWorker.java:728)
	at io.kestra.worker.DefaultWorker.handleTask(DefaultWorker.java:379)
	at io.kestra.worker.DefaultWorker.lambda$run$8(DefaultWorker.java:294)
	at io.micrometer.core.instrument.internal.TimedRunnable.run(TimedRunnable.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:104)
		... 18 more
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class software.amazon.awssdk.services.athena.model.Row and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77)
	at com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1359)
	at com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:415)
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.failForEmpty(UnknownSerializer.java:52)
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.serialize(UnknownSerializer.java:29)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:503)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:423)
	at com.fasterxml.jackson.databind.SequenceWriter.write(SequenceWriter.java:153)
	at io.kestra.core.utils.Rethrow.lambda$throwConsumer$0(Rethrow.java:49)
	at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:196)
	at reactor.core.publisher.FluxFilterFuseable$FilterFuseableSubscriber.tryOnNext(FluxFilterFuseable.java:148)
	at reactor.core.publisher.FluxIterable$IterableSubscriptionConditional.fastPath(FluxIterable.java:733)
	at reactor.core.publisher.FluxIterable$IterableSubscriptionConditional.request(FluxIterable.java:620)
	at reactor.core.publisher.FluxFilterFuseable$FilterFuseableSubscriber.request(FluxFilterFuseable.java:191)
	at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.request(FluxPeekFuseable.java:144)
	at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.request(FluxDoFinally.java:140)
	at reactor.core.publisher.Operators$BaseFluxToMonoOperator.request(Operators.java:2066)
	at reactor.core.publisher.BlockingSingleSubscriber.onSubscribe(BlockingSingleSubscriber.java:54)
	at reactor.core.publisher.Operators$BaseFluxToMonoOperator.onSubscribe(Operators.java:2050)
	at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onSubscribe(FluxDoFinally.java:107)
	at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onSubscribe(FluxPeekFuseable.java:178)
	at reactor.core.publisher.FluxFilterFuseable$FilterFuseableSubscriber.onSubscribe(FluxFilterFuseable.java:87)
	at reactor.core.publisher.FluxIterable.subscribe(FluxIterable.java:179)
	at reactor.core.publisher.FluxIterable.subscribe(FluxIterable.java:83)
	at reactor.core.publisher.Mono.subscribe(Mono.java:4576)
	at reactor.core.publisher.Mono.block(Mono.java:1778)
	... 17 more
```

### Environment

- Kestra Version: 1.1.0

--- a/src/main/java/io/kestra/plugin/aws/athena/Query.java
+++ b/src/main/java/io/kestra/plugin/aws/athena/Query.java
@@ -310,7 +310,8 @@ public class Query extends AbstractConnection implements RunnableTask<Query.Quer
         File tempFile = runContext.workingDir().createTempFile(".ion").toFile();
 
         try (var output = new BufferedWriter(new FileWriter(tempFile), FileSerde.BUFFER_SIZE)) {
-            Long count = FileSerde.writeAll(output, Flux.fromIterable(results)).block();
+            Long count = FileSerde.writeAll(output, Flux.fromIterable(results).mapNotNull(row -> map(columnInfo, row))).block();
+
             return Pair.of(
                 runContext.storage().putFile(tempFile),
                 count

--- a/src/test/java/io/kestra/plugin/aws/athena/QueryTest.java
+++ b/src/test/java/io/kestra/plugin/aws/athena/QueryTest.java
@@ -68,4 +68,28 @@ class QueryTest {
         assertThat(((Map<String, Object>) output.getRows().get(0)).get("map"), is("{bar=2, foo=1}"));
         assertThat(((Map<String, Object>) output.getRows().get(0)).get("timestamp"), is(LocalDateTime.parse("2008-09-15T03:04:05.324")));
     }
+
+    @Test
+    void runStore() throws Exception {
+        var runContext = runContextFactory.of();
+
+        var query = Query.builder()
+            .id("athena_store_test")
+            .type(Query.class.getName())
+            .region(Property.ofValue("eu-west-3"))
+            .accessKeyId(Property.ofValue(accessKey))
+            .secretKeyId(Property.ofValue(secretKey))
+            .database(Property.ofValue("units"))
+            .outputLocation(Property.ofValue("s3://kestra-unit-test"))
+            .query(Property.ofValue("select * from types"))
+            .fetchType(Property.ofValue(FetchType.STORE))
+            .skipHeader(Property.ofValue(true))
+            .build();
+
+        var output = query.run(runContext);
+
+        assertThat(output, notNullValue());
+        assertThat(output.getUri(), notNullValue());
+        assertThat(output.getRows(), is(nullValue()));
+    }
 }

